### ISSUE TITLE
can't detect track in video elements

### ISSUE THREAD

**Issue by goatandsheep** [maranran/eslint-plugin-vue-a11y#8]
I have a track in my video element and the captions work well, but it is not being detected by the linter

**Comment by maranran** [comment]
your code ? @goatandsheep

**Comment by goatandsheep** [comment]
```html
<video
    :ref="id"
    class="video-js vjs-default-skin"
  >
    <track
      :src="track"
    >
</video>
```
> eslint(vue-a11y/media-has-caption): Media elements such as \<audio> and \<video> must have a \<track> for captions.

**Comment by maranran** [comment]
The track element must be present for each HTML5 audio element, with kind="captions" set as a property.  [detail]( https://dequeuniversity.com/rules/axe/2.1/video-caption) @goatandsheep  you should add  kind="captions"  for  track element
